youtube_dl/extractor/mediaklikk.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_urllib_parse_unquote
+)
+
+
+class MediaKlikkIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mediaklikk\.hu/video/(?:[^/]+/)'
+    _TESTS = [
+        {
+            'url': 'https://www.mediaklikk.hu/video/az-evszakok-buvoleteben-osz/',
+            'info_dict': {
+                'id': '2512015',
+                'title': 'Az évszakok bűvöletében, Ősz',
+                'series': 'Az évszakok bűvöletében',
+                'ext': 'mp4'
+            }
+        },
+        {
+            'url': 'https://www.mediaklikk.hu/video/sporthirado-350-resz/',
+            'info_dict': {
+                'id': '2523053',
+                'title': 'Sporthíradó, 350. rész',
+                'series': 'Sporthíradó',
+                'ext': 'mp4'
+            }
+        },
+    ]
+
+    def _real_extract(self, url):
+        webpage = self._download_webpage(url,
+                                         None,
+                                         note='Fetching page')
+        pattern = r"mtva_player_manager\.player\(document.getElementById\(.*\),(.*)\);"
```

Comment:
Capturing empty string is senseless. If you expect a dict then capture group must contain `{}`.